src/components/stepper/bl-stepper-item.ts

```diff
@@ -0,0 +1,258 @@
+    this.onItemClick(this.id);
+  }
+
+  private handleKeyDown(event: KeyboardEvent) {
+    if (!this.isClickable) {
+      return;
+    }
+
+    if (event.key === "Enter" || event.key === " ") {
+      event.preventDefault();
+      this.onItemClick(this.id);
+    }
+  }
+
+  private handleMouseEnter() {
+    if (this.isClickable && this.variant === "default") {
+      this.variant = "hover";
+    }
+  }
+
+  private handleMouseLeave() {
+    if (this.variant === "hover") {
+      this.variant = "default";
+    }
+  }
+
+  render(): TemplateResult {
+    const classes = {
+      "stepper-item": true,
+      [`variant-${this.variant}`]: true,
+      [`type-${this.stepperType}`]: true,
+      [`direction-${this.direction}`]: true,
+      disabled: this.disabled,
+      clickable: this.isClickable,
+    };
+
+    const content = html`
+      <div class="stepper-content">
+        ${this.title ? html`<div class="stepper-title">${this.title}</div>` : ""}
+        ${this.description ? html`<div class="stepper-description">${this.description}</div>` : ""}
+      </div>
+    `;
+
+    const connectorState =
+      this.variant === "success" || this.variant === "active" ? "completed" : "default";
+
+    const stepIndicator = html`
+      <div class="connector-wrap">
+        ${this.direction === "horizontal"
+          ? html`
+              ${this.showLeadingConnector
+                ? html`<div class="connector connector-leading ${connectorState}"></div>`
+                : html`<div class="connector-placeholder"></div>`}
+              <div class="stepper-indicator">
+                ${this.shouldShowIcon
+                  ? html`<bl-icon name="${this.iconName}" class="step-icon"></bl-icon>`
+                  : this.stepperType === "number"
+                  ? html`<span class="step-number">${this.stepNumber}</span>`
+                  : ""}
+              </div>
+              ${this.showTrailingConnector
+                ? html`<div class="connector connector-trailing ${connectorState}"></div>`
+                : html`<div class="connector-placeholder"></div>`}
+            `
+          : html`
+              ${this.showLeadingConnector
+                ? html`<div class="connector connector-leading ${connectorState}"></div>`
+                : ""}
+              <div class="stepper-indicator">
+                ${this.shouldShowIcon
+                  ? html`<bl-icon name="${this.iconName}" class="step-icon"></bl-icon>`
+                  : this.stepperType === "number"
+                  ? html`<span class="step-number">${this.stepNumber}</span>`
+                  : ""}
+              </div>
+              ${this.showTrailingConnector
+                ? html`<div class="connector connector-trailing ${connectorState}"></div>`
+                : ""}
+            `}
+      </div>
+    `;
+
+    return html`
+      <div
+        class="${classMap(classes)}"
+        role="button"
+        tabindex="${this.isClickable ? "0" : "-1"}"
```

Comment:
I can land on the disabled steps by Tabbing, it shouldn't focus on disabled stepper items.